docs/pages/dual-signature-system.mdx

```diff
@@ -71,20 +71,38 @@ function getApprovalExpiry(
 
 ## Signatures
 
-Comments require different signatures depending on whether the author has granted approval to the app:
+Comments require different signatures depending on whether the author has granted approval to the app.
 
-**With Approval:** When a user has granted approval to an app, comments only need to be signed by the app. The app can post comments on the user's behalf using the pre-existing approval.
+### With Approval
 
-**Without Approval:** When no approval exists, comments must be signed by both the app and the comment author.
+When a user has granted approval to an app, the app can post comments on the user’s behalf. This is done using the `withSig` methods (e.g. `postCommentWithSig`), where:
 
-This means that every comment is attributable to an app and author. This enables the app to impose offchain limitations and gating on which comments it wants to co-sign, as well as allowing the app to choose whether to index and display comments from other apps or not.
+- The **author signature can be omitted** (pass an empty string).
+- The app signature is required, and the transaction can be sent by any relayer (this enables gasless UX) or the app itself.
+
+```typescript
+await postCommentWithSig({
+  author,
+  app,
+  content,
+  // authorSignature: "0x", // authorSignature omitted
```

Comment:
do this comment need to be here? I mean what is the purpose of having the parameter on that option object anyway? If you pass both app and author signature then what is happening, just normal postComment flow?